.github/workflows/qemu.yaml

```diff
@@ -0,0 +1,72 @@
+name: QEMU environment
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  release:
+    types:
+      - published
+
+jobs:
+  build-linux-arm64:
+    runs-on: ubuntu-22.04
+    steps:
```

Comment:
Do you know if we can use the githubToken thing to save the state of the image after we do all of the steps to prep for running the VM? 

```
# speeds up builds by storing container images in a GitHub package registry.
githubToken: ${{ github.token }}     
```
Wasn't sure if that's only used in that run-on-arch-action

Reply:
Specific to that action iinm